libs/providers/openshift.py

```
@@ -263,6 +263,24 @@ def vm_dict(
+
+        if result_vm_info["guest_agent_running"]:
+            try:
+                for sample in TimeoutSampler(
+                    wait_timeout=60,
+                    sleep=5,
+                    func=_all_interfaces_named,
+                ):
+                    if sample:
+                        break
+            except TimeoutExpiredError:
+                LOGGER.warning(f"Guest agent on VM {cnv_vm_name} did not report interfaceName for all NICs within 60s")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)